ingestion/justice_data_source/source.py

```diff
@@ -179,7 +179,14 @@ def _make_chart(
         )
 
         # add tag so entity displays in find-moj-data
-        tags = ["dc_display_in_catalogue", chart_data["domain"]]
+        # temporiliy add both old domain tag and new subject area tag
+        tags = [
+            "dc_display_in_catalogue",
+            chart_data["domain"],
+        ]
+        if domains_to_subject_areas.get(chart_data["domain"].lower()):
```

Comment:
minor code style thing but it would be nice to extract `domains_to_subject_areas.get(chart_data["domain"].lower()` to a variable here to avoid repeating it.

you could use the walrus operator:

```python
if subject_area := domains_to_subject_areas.get(chart_data["domain"].lower()):
  tags.append(subject_area)
```